client/lib/redux-bridge/index.js

```diff
@@ -25,3 +31,21 @@ export function reduxDispatch( ...args ) {
 	}
 	reduxStore.dispatch( ...args );
 }
+
+function markedFluxAction( action ) {
+	return Object.assign( {}, action, { type: `FLUX_${ action.type }` } );
+}
+
+// this is a Map<ActionType:string, transform:action=>action
+const forwardActionMap = new Map();
+forwardActionMap.set( 'RECEIVE_MEDIA_ITEM', markedFluxAction );
+forwardActionMap.set( 'RECEIVE_MEDIA_ITEMS', markedFluxAction );
```

Comment:
Will we put all the forwarder definitions here in this file?

Reply:
not sure. we could, or we could make it a registry?

with a registry my worry would be > 1 place trying to register with different transforms. 

which makes me think we don't want to allow transforms?